webapp/src/components/link_tooltip/tooltip.css

```diff
@@ -115,8 +115,10 @@
     padding: 0 5px;
     font: .75em/2 SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace;
     color: var(--blue);
-    white-space: nowrap;
     background-color: var(--light-blue);
     border-radius: 3px;
-    maxWidth: 140px;
+    max-width: 140px;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
```

Comment:
@asaadmahmood This is how the long branch names look now. Please let me know if this is fine.

![Screenshot from 2020-06-08 10-55-55](https://user-images.githubusercontent.com/16541325/84018471-609c8580-a980-11ea-880f-711c6c1b2d5b.png)

Reply:
@hanzei Looks good. The description text however looks too light.

Reply:
@asaadmahmood sure. That issue is covered by https://github.com/mattermost/mattermost-plugin-github/issues/294